### ISSUE TITLE
So we can not change the word embedding with the pretrained LM?

### ISSUE THREAD

**Issue by herbertchen1** [huggingface/pytorch-openai-transformer-lm#34]
And training the LM is very hard...?

**Comment by rodgzilla** [comment]
I do not think that you can change the word embedding easily since its dimension must be the same as the output of each layer, in the case of the pre-trained model `768` (`cfg.n_embd`).
